jobs/kube-controller-manager/templates/config/bpm.yml.erb

```diff
@@ -15,7 +15,7 @@ processes:
   - --leader-elect
   - --root-ca-file=/var/vcap/jobs/kube-controller-manager/config/ca.pem
   - --service-account-private-key-file=/var/vcap/jobs/kube-controller-manager/config/service-account-private-key.pem
-  - --service-cluster-ip-range=10.100.200.0/24
```

Comment:
I am no sure if we need that at all.
According to the doc 
```
--cluster-cidr string-
  | CIDR Range for Pods in cluster. Requires --allocate-node-cidrs to be true
```
and we are not using the allocate-node-cidrs=true

Reply:
--service-cluster-ip-range is needed, it sets the address block for hosting kubernetes service IPs, if not set, kubernetes uses a 10.X.X.X default of some sort, which could potentially clash with users pod IP space, or even network routable IP space.

Reply:
Ah, thats for the controller manager and not the apiserver, yeah you are correct, that should be fine to drop for now.